main/integration-tests/support/resources/users/users.js

```diff
@@ -54,6 +54,10 @@ class Users extends CollectionResource {
     };
   }
 
+  async cleanup(user) {
```

Comment:
I am thinking of adding this as the default implementation in Resource. So, we might be able to remove it from here.

Reply:
I'm leaning towards keeping it here as a temporary before we have the default implementation. I had 100+ users created in my environment yesterday when running integ tests, having it here saves a lot of trouble.